packages/wallet/test/services/WalletUtil.test.ts

```diff
@@ -122,6 +128,31 @@ describe('WalletUtil', () => {
         value: { coins: 150_000_000n }
       });
     });
+
+    it('resolveInput resolves inputs from provided signed transactions utxo set', async () => {
+      const sigendTxs = mocks.queryTransactionsResult.pageResults.map(toSignedTx);
+
+      const resolver = createInputResolver({
+        transactions: { outgoing: { signed$: of(sigendTxs) } },
+        utxo: { available$: of() }
+      });
+      expect(
+        await resolver.resolveInput({
+          index: 0,
+          txId: Cardano.TransactionId('12fa9af65e21b36ec4dc4cbce478e911d52585adb46f2b4fe3d6563e7ee5a61a')
```

Comment:
I think it would be better to pick assertion data from signedTxs, so that the test is more resilient to refactors/test data changes, and also the intent of the assertion is easier to understand
```suggestion
          txId: signedTxs[0].id
```

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/1113/commits/7761eaaca8cd66a6a67e70588333ea1ec10da27a